apps/spectator/src/components/PopupDialog/index.tsx

```
@@ -0,0 +1,75 @@
+              src={imageUrl}
+              alt="2025 외대월드컵 팝업 이미지"
+              width={0}
+              height={0}
+              sizes="100vw"
+              draggable={false}
+              priority
+            />
+          </div>
+          <div className={styles.buttonContainer}>
+            <button className={styles.button} onClick={handleDontShowToday}>
+              하루동안 보지 않기
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)